cas-server-support-osf/src/main/java/io/cos/cas/authentication/handler/support/OpenScienceFrameworkPrincipalFromRequestRemoteUserNonInteractiveCredentialsAction.java

```diff
@@ -113,21 +128,21 @@ public PrincipalAuthenticationResult(final String username, final String institu
             this.institutionId = institutionId;
         }
 
-        /**
-         * @return the username.
-         */
         public String getUsername() {
             return username;
         }
 
-        /**
-         * @return the institutionId.
-         */
         public String getInstitutionId() {
             return institutionId;
         }
     }
 
+    /** Authentication Delegation Protocol: CAS by pac4j. */
+    public static final String PROTOCOL_CAS = OpenScienceFrameworkInstitution.DelegationProtocols.CAS_PAC4J.name();
```

Comment:
Uses `enum` instead of literal `String` for `PROTOCOL_CAS` (above) and same for `PROTOCOL_SAML` (below).